### ISSUE TITLE
Output of HLAminer_HPRA.csv is empty

### ISSUE THREAD

**Issue by tao840813** [bcgsc/HLAminer#26]
Sorry for dumb question, but I did two command for HLA classification on my own fastq file 

./minimap2-2.24_x64-linux/minimap2 -t 60 -ax map-ont --MD ./database/GCA_000001405.15_GRCh38_genomic.chr-only-noChr6-HLA-I_II_GEN.fa.gz TESLA_86_1.fastq.gz | ./bin/HLAminer.pl -h ./database/HLA-I_II_GEN.fasta -s 500 -q 1 -i 1 -p ./database/hla_nom_p.txt -a stream
and 
./bin/HLAminer.pl -a TESLA_86.sam -h ./database/HLA-I_II_GEN.fasta -p ./database/hla_nom_p.txt -s 500 -q 1 -i 1

Both command leads to empty prediction. Any solution of this situation or I just do a  wrong command?
![2022-11-10_17-58](https://user-images.githubusercontent.com/12207976/201060306-f5d1bce5-966f-43d7-961c-bdf0cd67fa8d.png)

BTW, the output of HLAminer_HPRA.log is different for two inputs(minimap steam and directly sam input). 
I saw issue like this , I am not sure if this is helpful
https://github.com/bcgsc/HLAminer/issues/12

sam
![2022-11-10_18-06_1](https://user-images.githubusercontent.com/12207976/201062479-657abe66-d974-44dc-9259-163fa9e710bb.png)
minimap2
![2022-11-10_18-06](https://user-images.githubusercontent.com/12207976/201062561-c332c679-e11b-4b0a-b0f5-98f821306164.png)

**Comment by warrenlr** [comment]
you could try lowering the score (-s) ?

Also, what is your read depth? 

I'm curious, did you try the streaming demo?  If the demo works, then it is something to do with your reads (depth, base accuracy, etc.)

**Comment by tao840813** [comment]
Sure, I tried lower score(-s) to 250 but same result. 
Average read depth calculated by samtools and bam file is 39.1491. (Does HLAminer have depth read limitation?)

I am also downloading ERR2585115.fastq.gz (extremly slow....), I will inform you when the result comes out.

**Comment by warrenlr** [comment]
no read depth limits, I was able to get good predictions on a 10X dataset (https://arxiv.org/pdf/2209.09155.pdf), but just curious about yours.

The preprint lists other data you could try

**Comment by tao840813** [comment]
Sorry for late responding. I find out that I forgot read2 file. So I added read2 file and tried TASR pipeline so that everything worked  well eventually. It is too stupid,  I felt embarrass to response. 
However, may I ask ideal computer spec and average running time when people predict HLA-II type by referencing HLA-I_II_GEN/CDS.fasta in TASR pipeline? And it is possible to run TASR with multithread? It only run 5% per days for my computer. I  understand that HLA-I_II_GEN/CDS.fasta are way bigger than HLA-ABC_GEN/CDS.fasta. But it does take way more time than I imaged.(Obviously, I can reference DMA_gen.fasta,DMB_gen.fasta ...etc separately, but I am still wondering)

Size:
    Read1:90070116
    Read2:90070116
My spec:
   CPU:AMD Ryzen 7 5800X 8-Core Processor
   MB:ROG STRIX X570-F GAMING
   Mem:32Gb =DIMM DDR4 8Gb*4
